src/main/java/org/ngafid/routes/PostMonthlyEventCounts.java

```diff
@@ -71,6 +71,8 @@ public Object handle(Request request, Response response) {
                 return gson.toJson(map.get(eventName));
             }
         } catch (SQLException e) {
+            LOG.severe(e.toString());
+            e.printStackTrace();
```

Comment:
I figure you were trying to debug here -- I have the fix for this implemented on another branch. currently gated ofc by my reword but since its just a change to the queries we can pull it in.

Basically, I totally forgot to fix some queries and also forgot to delete event_statistics table.